django/contrib/postgres/fields/ranges.py

```diff
@@ -41,7 +42,7 @@ def value_to_string(self, obj):
             "lower": value.lower,
```

Comment:
Sounds good; I've just tried implementing this, and the only change is that in e.g. IntegerRangeField, the lower/upper are now strings rather than integers in the JSON, which does match ArrayField, so is presumably okay?